measures/ApplyUpgrade/measure.rb

```diff
@@ -504,10 +504,7 @@ def set_battery(measures, hpxml)
     end
   end
 
-  def set_electric_panel(measures, hpxml_bldg, upgrade_args_hash)
-    measures['ResStockArgumentsPostHPXML'][0]['electric_panel_service_max_current_rating'] = hpxml_bldg.electric_panels[0].max_current_rating
-    measures['ResStockArgumentsPostHPXML'][0]['electric_panel_breaker_spaces_rated_total'] = hpxml_bldg.electric_panels[0].breaker_spaces_total
-
+  def set_electric_panel(measures, _hpxml_bldg, upgrade_args_hash)
     panel_system_additions = get_panel_system_additions(upgrade_args_hash)
     measures['ResStockArgumentsPostHPXML'][0].update(panel_system_additions)
   end
```

Comment:
It'd obviously be nice if this logic could be moved to ResStockArgumentsPostHPXML. Especially since [the existing logic doesn't work very well](https://github.com/NREL/resstock/issues/1517). But I understand that might be beyond the scope of what you want to do.